### PR TITLE
Mas fixes

### PR DESCRIPTION
**Pull request by TomDijkema**
Fixes for scheduling MAS.
Completion message should only appear with a successful POST.
Allow batching is now applied to every single MAS, instead of them all at the same time.

### CODE REVIEW

File: src/api/specimen/ScheduleSpecimenMAS.ts [hunk 1]
```diff
@@ -32,6 +29,8 @@ const ScheduleSpecimenMAS = async (handle: string, MASRequest: Dict, batching: b
             specimenMAS = data.data[0].attributes;
         } catch (error) {
             console.warn(error);
+
```
Comment (southeo) [inline src/api/specimen/ScheduleSpecimenMAS.ts:32]: remove space

**Review by southeo — APPROVED** [review]
:cookie:

File: src/api/specimen/ScheduleSpecimenMAS.ts (hunk 1 above)
Comment (TomDijkema) [reply src/api/specimen/ScheduleSpecimenMAS.ts:32]: You mean the enter? That is intentional

Comment (southeo) [reply src/api/specimen/ScheduleSpecimenMAS.ts:32]: alright well **I** would remove it :laughing: java conventions are different i guess

**Review by samleeflang — APPROVED** [review]

**Pull request merged**